### PR TITLE
Conseil #132 - sbt tasks for running subprojects

### PR DESCRIPTION
**Pull request by piotrkosecki**
I've added 3 sbt tasks for easier running of the project. Also updated the readme showing how my improvement can be used. 

Improves #132

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,15 @@ Run locally using SBT:
...
 
+Or you can use the predefined tasks:
+- for Conseil:
+`sbt -Dconfig.file="{path to custom config file}" runConseil`
+- for Lorre 
+`sbt -Dconfig.file="{path to custom config file}" "runLorre alphanet"`
+- for schema generation
+`sbt -Dconfig.file="{path to custom config file}" genSchema`
+
+
```
Comment (ivanopagano) [inline README.md]: I would explicitly state here that using the predefined tasks you get a custom-tailored vm launch configuration that's best suited for the actual task

File: build.sbt [hunk 2]
```diff
@@ -38,4 +38,22 @@ excludeDependencies ++= Seq(
...
+
+fork in runConseil := true
+lazy val runConseil = taskKey[Unit]("A conseil run task.")
+javaOptions in runConseil ++= Seq("-Xms512M", "-Xmx4096M", "-Xss1M", "-XX:+CMSClassUnloadingEnabled")
+fullRunTask(runConseil, Runtime, "tech.cryptonomic.conseil.Conseil")
+
+fork in runLorre := true
+lazy val runLorre = inputKey[Unit]("A lorre run task.")
+javaOptions ++= Seq("-Xmx512M", "-Xss1M", "-XX:+CMSClassUnloadingEnabled")
+runLorre := Def.inputTaskDyn {
```
Comment (ivanopagano) [inline build.sbt]: Don't know in detail how this works, but from the [docs](https://www.scala-sbt.org/1.0/docs/Faq.html#How+can+I+create+a+custom+run+task%2C+in+addition+to+%3F) I would've thought that we only needed to use `fullRunInputTask` to get the cli argument.

Why the dynamic input part?

Comment (piotrkosecki) [reply build.sbt]: yeah, that would seem natural, but its sbt so just using `fullRunInputTask` does not want to read the config file 😄

Comment (piotrkosecki) [reply build.sbt]: dynamic part basically allows to evaluate args and put them into the task. In regular input task you could only hardcode things.

File: README.md (hunk 1 above)
Comment (piotrkosecki) [reply README.md]: ok, quoted you in the docs 😄

**Pull request merged**